week04/style.css

```
@@ -0,0 +1,96 @@
+
+
+html,body{
+    height: 99%;
+    /*왜 100%로 하면 스크롤바가 생기는지?*/
+    background-color: #c1bcea;
+}
+.top_end{
+    height: 10%;
+    /*border: 2px solid blue;*/
+}
+.background{
+    display: flex;
+    flex : 1;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    /* border: 2px solid black; */
+    height: 80%;
+    justify-content: space-around;
+    
+}
+
+.top{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    /* border: 2px solid red; */
+    height: 20%;
+    width: 30%;
+    
+    
+}
+
+.score{
+    color: white;
+    width: 40%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    /* border: 2px solid green; */
+    font-size: 45px;
+
+}
+
+
+
+.text{
+    /* border: 2px solid yellow; */
+    color: white;
+    height: 30%;
+    display: flex;
+    align-items: end;
+} 
+.middle{
+    display: flex;
+    justify-content: space-around;
+    /* border: 2px solid orange; */
+    height: 35%;
+    width: 70%;
+}
+
+/* #human_select{
+    border: 2px solid blue;
+} */
+
+/* #robot_select{
+} */
+.bottom{
+    display: flex;
+    justify-content: space-around;
+    /* border: 2px solid yellow; */
+    height: 10%;
+    width: 50%;
+}
+
+#img_toScissor,#img_toRrock,#img_toPaper{
+    width: 20%;
+    font-size: 20px;
+    border:0;
+    border-radius: 15%;
+}
```

여러 요소에 대해 CSS를 적용할 때는 각 요소의 id를 단순 나열하는 것보다 공통점을 찾는 것이 코드의 가독성을 높입니다. class의 이름은 여러 요소에 중복될 수 있으므로 각 요소에 같은 class 이름을 부여하여 CSS를 적용하거나, 이 경우 `.option`의 자식 `button`에 대한 공통 사항이므로 다음과 같이 작성할 수 있습니다.
```suggestion
body .option button {
    width: 20%;
    font-size: 20px;
    border:0;
    border-radius: 15%;
}
```